sorting/Topological_sort.cpp

```diff
@@ -0,0 +1,91 @@
+#include <algorithm>
+#include <iostream>
+using namespace std;
+
+#define MAX 10
+int n, adj[MAX][MAX];
+int front = -1, rear = -1, Queue[MAX];
+
+void CreateGraph() {
+	int i, max_edge, origin, dest;
+	cout<<"Enter number of Vertex\n";
+	cin>>n;
+	max_edge = n*(n-1);
+	for(int i = 0; i <= max_edge;i ++) {
+		cout<<"Enter Edge Source and Destination 0 to exit\n";
+		cin>>origin>>dest;
+		if(origin == 0 && dest == 0)
+		break;

```

Comment:
Please make it look like C++ code, not C
```suggestion
		if (origin == 0 && dest == 0) {
		    break;
		}

```
Same in other parts of the code.